src/main/java/dan200/computercraft/client/gui/GuiPrintout.java

```diff
@@ -204,7 +205,7 @@ public void drawScreen(int mouseX, int mouseY, float f)
             int lineIdx = ItemPrintout.LINES_PER_PAGE * m_page + line;
             if( lineIdx >= 0 && lineIdx < m_text.length )
             {
-                fontRenderer.drawString( m_text[lineIdx], x, y, m_colours[lineIdx], null, 0, 0, false );
+                fontRenderer.drawString( m_text[lineIdx], x, y, m_colours[lineIdx], null, 0, 0, false, new Palette() );
```

Comment:
Instead of creating this every frame, could we cache it? Just store a static Palette.DEFAULT member with the default colour values